src/ngx_http_lua_module.c

```diff
@@ -241,7 +241,7 @@ static ngx_command_t ngx_http_lua_cmds[] = {
       ngx_http_lua_rewrite_by_lua,
       NGX_HTTP_LOC_CONF_OFFSET,
       0,
-      (void *) ngx_http_lua_rewrite_handler_inline },
+      (void *) ngx_http_lua_rewrite_handler_sets },
```

Comment:
The `_sets` suffix looks strange. What does it mean? Settings? But "sets" are very different from "settings", I'm afraid.